EXILED/Exiled.Events/Patches/Events/Player/InteractingDoor.cs

```diff
@@ -65,8 +66,14 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     new(OpCodes.Dup),
                     new(OpCodes.Call, Method(typeof(Handlers.Player), nameof(Handlers.Player.OnInteractingDoor))),
 
+                    // if (!ev.CanInteract)
+                    //    go to retLabel;
+                    new(OpCodes.Callvirt, PropertyGetter(typeof(InteractingDoorEventArgs), nameof(InteractingDoorEventArgs.CanInteract))),
+                    new(OpCodes.Brfalse_S, retLabel),
+
                     // if (ev.IsAllowed)
                     //    go to interactionAllowed;
+                    new(OpCodes.Ldloc_S, ev.LocalIndex),
```

Comment:
Maybe call `Dup` again? Looks not really good when you do like this

Reply:
It's can't call dup because there br false right before and it's would be needed to pop so it's better like this